config/config_soft_filer_somatic.yaml

```diff
@@ -0,0 +1,21 @@
+filters:
+  depth:
+    description: "Soft filter on depth"
+    expression: "FORMAT:DP:200"
+    soft_filter: "True"
+  ad:
+    description: "Soft filter variants with few observations (AD lower than 20)"
+    expression: "(FORMAT:AD:1 < 20)"
+    soft_filter: "True"
+  vaf:
+    description: "Soft filter variants with low vaf (AF lower than 0.01)"
+    expression: "(INFO:AF:0 < 0.01)"
+    soft_filter: "True"
+  intron:
+    description: "Soft filter intronic variants except if also splice, in cosmic, or in gata2 or tert genes"
+    expression: "(exist[intron_variant, VEP:Consequence] and !exist[splice, VEP:Consequence] and VEP:SYMBOL != TERT and VEP:SYMBOL != GATA2 and !exist[COSV[0-9]+, VEP:Existing_variation])"
```

Comment:
In Uppsala we also report an intron region b/w exon 4-5 for GATA2 so added that.